mne/preprocessing/tests/test_maxwell.py

```diff
@@ -0,0 +1,81 @@
+# Author: Mark Wronkiewicz <wronk@uw.edu>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import warnings
+import numpy as np
+from numpy.testing import (assert_equal, assert_allclose,
+                           assert_array_almost_equal)
+from scipy.special import sph_harm as scipy_sph_harm
+from mne.preprocessing import maxwell
+from mne.datasets import sample
+from mne.io import Raw
+from mne.datasets import testing
+
+warnings.simplefilter('always')  # Always throw warnings
+
+
+def test_spherical_harmonic():
+    """Test for spherical harmonics"""
+    deg = 1
+    order = -1
+    azimuth = np.random.random_sample(size=(50, 1)) * 2 * np.pi
+    polar = np.random.random_sample(size=(50, 1)) * np.pi
+
+    # Internal calculation
+    sph_harmonic = maxwell._sph_harmonic(deg, order, azimuth, polar)
+    # Check against scipy
+    sph_harmonic_scipy = scipy_sph_harm(order, deg, azimuth, polar)
+
+    assert_array_almost_equal(sph_harmonic, sph_harmonic_scipy, decimal=15,
+                              err_msg='Spherical harmonic mismatch')
+
+
+@testing.requires_testing_data
+def test_maxwell_filter():
+    """Test multipolar moment and Maxwell filter"""
+
+    # Load data
+    data_path = sample.data_path()
```

Comment:
Do not use `sample.data_path` -- this causes downloading of the 1+ GB `sample` data path instead of the `testing` data. I think you meant `testing.data_path()`. And shouldn't you be using files from `SSS/`, not `MEG/sample`?